api/tests/routes/public/collective/endpoints/bookings_test.py

```diff
@@ -34,7 +33,7 @@ def test_cancel_collective_booking(self, client, venue):
         db.session.refresh(booking)
 
         assert booking.cancellationReason == educational_models.CollectiveBookingCancellationReasons.PUBLIC_API
-        assert booking.cancellationDate == datetime(2023, 1, 1, 10)
+        assert format_datetime(booking.cancellationDate) == format_datetime(datetime.utcnow())
```

Comment:
Au lieu de comparer les représentations textuelles approximatives de ces 2 objets datetime, tu pourrais comparer les objets eux-mêmes. Il existe la fonction `pytest.approx()` pour ça. Tu devrais trouver dans notre base de code quelques exemples d'utilisation.

Reply:
Alors en effet il y a eu l'inversion du timespecs="seconds") et ca fonctionne maintenant, par contre avec pytest.approx() cela ne fonctionne pas partout donc pour le mooment je reste avec la fonction

Reply:
@borisLeBonPro : Quelle erreur as-tu ? Je réussis à faire passer ce test comme suit : 

```suggestion
        assert booking.cancellationDate.timestamp() == pytest.approx(datetime.utcnow().timestamp(), rel=1)
```

Reply:
En effet cela fonctionne ainsi, je vais voir a l'appliquer sur d'autres test, merci :)